src/HttpClientInterception/HttpRequestInterceptionBuilderExtensions.cs

```diff
@@ -305,7 +305,7 @@ public static HttpRequestInterceptionBuilder RegisterWith(this HttpRequestInterc
 
             if (predicate == null)
             {
-                return builder.ForContent(null as Func<HttpContent, Task<bool>>);
+                return builder.ForContent(null);
```

Comment:
Without having the IDE open, I can't remember whether I did this to force it through a specific overload for some reason.

Will refresh my memory later on.

Reply:
Can't remember why that was like that.